packages/playground/src/components/dapp-container/dapp-container.tsx

```diff
@@ -52,17 +52,13 @@ export class DappContainer {
     this.frameWindow = iframe.contentWindow as Window;
     this.$onMessage = this.configureMessageChannel.bind(this);
```

Comment:
why add `$`?

Reply:
Just a visual cue. We can drop it, I don't mind.